pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go

```diff
@@ -123,54 +123,8 @@ func (pl *RequestedToCapacityRatio) ScoreExtensions() framework.ScoreExtensions
 	return nil
 }
 
-func validateFunctionShape(shape functionShape) error {
-	if len(shape) == 0 {
-		return fmt.Errorf("at least one point must be specified")
-	}
-
-	for i := 1; i < len(shape); i++ {
-		if shape[i-1].utilization >= shape[i].utilization {
-			return fmt.Errorf("utilization values must be sorted. Utilization[%d]==%d >= Utilization[%d]==%d", i-1, shape[i-1].utilization, i, shape[i].utilization)
-		}
-	}
-
-	for i, point := range shape {
-		if point.utilization < minUtilization {
-			return fmt.Errorf("utilization values must not be less than %d. Utilization[%d]==%d", minUtilization, i, point.utilization)
-		}
-		if point.utilization > maxUtilization {
-			return fmt.Errorf("utilization values must not be greater than %d. Utilization[%d]==%d", maxUtilization, i, point.utilization)
-		}
-		if point.score < minScore {
-			return fmt.Errorf("score values must not be less than %d. Score[%d]==%d", minScore, i, point.score)
-		}
-		if int64(point.score) > maxScore {
```

Comment:
Please note that the validation of the function shape was incorrectly reporting values exceeding maximum, because the validation was performed after scaling of the values passed in the config.

The allowed range is [1-10]. If the value was exceeded the max, it was reporting the scaled value in the error message.

Reply:
ack